wg-naming/recommendations/002-blacklist-whitelist-allowlist-denylist.md

```diff
@@ -0,0 +1,95 @@
+# Recommendation: replace blacklist/whitelist with allowlist/denylist
+
+**Last Updated**: 2020-12-01
+
+**Status:** Accepted
+
+## Suggested Alternatives 
+
+Replacements: 
+- `allowlist/denylist`
+
+## Context
+
+The underlying assumption of the whitelist/blacklist metaphor is that white = good and black = bad. Because colors in and of themselves have no predetermined meaning, any meaning we assign to them is cultural: for example, the color red in many Southeast Asian countries is lucky, and is often associated with events like marriages, whereas the color white carries the same connotations in many European countries. In the case of whitelist/blacklist, the terms originate in the publishing industry – one dominated by America and England, two countries which participated in slavery and which grapple with their racist legacies to this day.
```

Comment:
Grammar nits
```suggestion
The underlying assumption of the whitelist/blacklist metaphor is that white = good and black = bad. Because colors in and of themselves have no predetermined meaning, any meaning we assign to them is cultural: for example, the color red in many Southeast Asian countries is lucky, and is often associated with events like marriages, whereas the color white carries the same connotations in many European countries. In the case of whitelist/blacklist, the terms originate in the publishing industry, which was historically dominated by America and England, two countries that participated in slavery and which grapple with their racist legacies to this day.
```